### ISSUE TITLE
mangoTheCat/goodpractice functionality

### ISSUE THREAD

**Issue by stephlocke** [lockedata/PackageReviewR#4]
How much of the functionality could be leveraged from `goodpractice`?
https://github.com/MangoTheCat/goodpractice

**Comment by noamross** [comment]
I think a great many things one might check for (e.g. paths constructed by `paste`), could be added as custom lintr or goodpractice extensions.

**Comment by stephlocke** [comment]
Yeah - I like goodpractice and I'd like to see it extended. There's thing like package dependencies and other things that fit in a review but for coding standards goodpractices seems ideal
